### ISSUE TITLE
Maintained

### ISSUE THREAD

**Issue by arb** [tommy351/sprite.styl#2]
@tommy351 are you still actively maintaining this project? I started using it yesterday and it works great. I ask because I was going to make PRs to update the documentation as I was having issues and the docs were missing some important information for first time users.

**Comment by tommy351** [comment]
No. This project has not been updated for a while. If you're interested, I can add you as a collaborator.

**Comment by arb** [comment]
Is there something else you would suggest using? This seems like a fine replacement for compass...
